environments/prod/backend_lb_config.yaml

```diff
@@ -305,10 +305,6 @@ gateways:
       # I&A
       - product: ia
         component: bail-case-api
-      
-      # CNP
-      - product: plum
```

Comment:
I would probably remove this in a separate PR so you don't have any race conditions where this PR tries to create the record before it's deleted

Reply:
OK sure, I'll sort that out now, thanks

Reply:
removed in - https://github.com/hmcts/azure-platform-terraform/pull/1273